### ISSUE TITLE
Question about defining Keycloak attributes using Annotations

### ISSUE THREAD

**Issue by jhobeck** [ch4mpy/spring-addons#10]
In my Keycloak instance I have defined a couple of attributes (x_id, y_id) that get passed in the token, I can retrieve these attributes from a Map on the token (other claims):
```
// Where authentication is KeycloakAuthenticationToken
KeycloakPrincipal principal = (KeycloakPrincipal<KeycloakSecurityContext>)authentication.getPrincipal();
Map<String, Object> attrs = getPrincipal().getKeycloakSecurityContext().getToken().getOtherClaims();
Object xAttr = attrs.get("x_id");
Object yAttr = attrs.get("y_id");
```

I tried using privateClaims with a single value, but this didn't seem to work.
```
@WithMockKeycloakAuth(
    authorities = { "ROLE_USER" },
        oidc = @OidcStandardClaims(
        email = "foo@bar.com",
        preferredUsername = "foo"),
        privateClaims = @ClaimSet(stringClaims = @StringClaim(name = "x_id", value = "1")))
```

I'm pretty sure this is obvious, I'm just missing something.  Saw a closed question about using @WithAccessToken but I could not find that annotation, I also didn't see anything under the accessToken parameter.
Using Keycloak 9.0.2, Java 11, SpringBoot 2.3.1, spring-security-oauth2-test-webmvc-addons 2.3.1

**Comment by ch4mpy** [comment]
I forgot to map `otherClaims` in authentication builder. Sorry.

Fixed with commit f26a20dbb3096449f2a156cffc60f45f28f460f0 which I'm currently releasing.

Your test should work as expected from 2.3.2 on.

**Comment by ch4mpy** [comment]
P.S.
For usability reasons, I dropped the separate definitions for `AccessToken` and `IDToken`.  I have no knowledge of cases where this was useful and as `AccessToken extends IDToken`, it's way easier to define values only once, even if it prevents from having different values in each (but again, I can't figure out why this would be useful).

As a consequence, `@WithAccessToken` disappeared in `2.3.0` (along with `@WithIDToken`). I preferred to reuse `@OidcStandardClaims` and `@IdTokenClaims` I already wrote for `@WithMockOidcId` (less code to maintain) to define OIDC claims and build both access and ID tokens out of it.
